imgui/core.pyx

```diff
@@ -4494,6 +4515,103 @@ def is_mouse_hovering_any_window():
     return cimgui.IsMouseHoveringAnyWindow()
 
 
+def is_mouse_hovering_rect(
+    float r_min_x, float r_min_y,
+    float r_max_x, float r_max_y,
+    bool clip=True
+):
+    """Test if mouse is hovering rectangle with given coordinates.
+
+    Args:
+        r_min_x, r_min_y (float): x,y coordinate of the upper-left corner
+        r_max_x, r_max_y (float): x,y coordinate of the lower-right corner
+
+    Returns:
+        bool: True if mouse is hovering the rectangle.
+
+    .. wraps::
+        bool IsMouseHoveringRect(
+            const ImVec2& r_min,
+            const ImVec2& r_max,
+            bool clip = true
+        )
+    """
+    return cimgui.IsMouseHoveringRect(
+        _cast_args_ImVec2(r_min_x, r_min_y),
+        _cast_args_ImVec2(r_max_x, r_max_y),
+        clip
+    )
+
+
+def is_mouse_dragging(int button = 0, float lock_threshold = -1.0):
+    """Returns if mouse is dragging.
+
+    Args:
+        button (int): mouse button index.
+        lock_threshold (float): if less than -1.0
+            uses io.MouseDraggingThreshold.
+
+    Returns:
+        bool: if mouse is dragging.
+
+    .. wraps::
+        bool IsMouseDragging(int button = 0, float lock_threshold = -1.0f)
+    """
+    return cimgui.IsMouseDragging(button, lock_threshold)
+
+
+def get_mouse_drag_delta(int button=0, float lock_threshold = -1.0):
+    """Dragging amount since clicking.
+
+    Args:
+        button (int): mouse button index.
+        lock_threshold (float): if less than -1.0
+            uses io.MouseDraggingThreshold.
+
+    Returns:
+        Vec2: mouse position two-tuple ``(x, y)``
+
+    .. wraps::
+        ImVec2 GetMouseDragDelta(int button = 0, float lock_threshold = -1.0f)
+    """
+    return _cast_ImVec2_tuple(
+        cimgui.GetMouseDragDelta(button, lock_threshold)
+    )
+
+
+def reset_mouse_drag_delta(int button = 0):
+    """Reset the mouse dragging delta.
+
+    Args:
+        button (int): mouse button index.
+
+    .. wraps::
+        void ResetMouseDragDelta(int button = 0)
+    """
+    cimgui.ResetMouseDragDelta(button)
+
+
+def get_mouse_cursor():
+    """Return the mouse cursor id.
+
+    .. wraps::
+        void GetMouseCursor()
+    """
+    return cimgui.GetMouseCursor()
+
+
+def set_mouse_cursor(cimgui.ImGuiMouseCursor type):
```

Comment:
Type is a built-in function in Python. It would be better to use other name e.g. `cursor/cursor_type` or use the `builtin_/keyword_` convention instead of plain `builtin/keyword`. It's up to you which one you will choose.